packages/rocketchat-ldap/server/loginHandler.js

```diff
@@ -100,13 +104,13 @@ Accounts.registerLoginHandler("ldap", function(loginRequest) {
 	if (user) {
 		if (user.ldap !== true) {
 			logger.info('User exists without "ldap: true"');
-			throw new Meteor.Error("LDAP-login-error", "LDAP Authentication succeded, but there's already an existing user with provided username ["+username+"] in Mongo.");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["Meteor" is not defined.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2313313427/issues/source?bid=3199634&fileBranchId=3199636#l107)